app/Makefile

```
@@ -0,0 +1,71 @@
+	cd "$(APP_DIR)"
+	$(DOCKER) build -f docker/Dockerfile.client -t "$(FRONTEND_IMAGE)" .
+
+docker-build-server: ## Build the backend image
+	cd "$(APP_DIR)"
+	$(DOCKER) build -f docker/Dockerfile.server -t "$(BACKEND_IMAGE)" .
+
+docker-build-worker: ## Build the worker image
+	cd "$(APP_DIR)"
+	$(DOCKER) build -f docker/Dockerfile.worker -t "$(WORKER_IMAGE)" .
+
+.PHONY: docker-up docker-down docker-logs
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)